README.md

```diff
@@ -55,6 +55,17 @@ For example:
 katex.render("c = \\pm\\sqrt{a^2 + b^2}", element, { displayMode: true });
 ```
 
+Note that the `throwOnError` option only deals with how TeX related parsing errors are handled.
```

Comment:
Would it be better to say something such as:  "Note that the `throwOnError` option only deals with how missing TeX commands are handled by the KaTeX parser. Other invalid input such as mismatched braces will always throw exceptions regardless of the `throwOnError` setting."